src/plugins/vis_builder/public/application/components/data_tab/field_details.tsx

```diff
@@ -53,7 +53,7 @@ export function FieldDetailsView({ field, details }: FieldDetailsProps) {
 
   return (
     <>
-      <EuiPopoverTitle>{title}</EuiPopoverTitle>
+      <EuiPopoverTitle tabIndex={0}>{title}</EuiPopoverTitle>
```

Comment:
does this apply both in discover and visbuilder?

Reply:
Just visBuilder, the tooltip PR has the discover tooltip

Reply:
I'm talking specifically about tabindex - I don't seem to see it in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7231/files?

Reply:
I had that in a separate PR, was going to raise after I merge the tooltip PR. Can add it now